Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -56,27 +55,23 @@ open class TabToolbarHelper: NSObject {
     let ImageStop = UIImage.templateImageNamed("nav-stop")
     let ImageSearch = UIImage.templateImageNamed("search")
     let ImageNewTab = UIImage.templateImageNamed("nav-add")
+    let ImageHome = UIImage(named: "menu-Home")
     
     func setMiddleButtonState(_ state: MiddleButtonState) {
-        middleButtonState = state
         switch state {
-            case .reload:
-                toolbar.multiStateButton.setImage(ImageReload, for: .normal)
-                toolbar.multiStateButton.accessibilityLabel = .TabToolbarReloadAccessibilityLabel
-            case .stop:
-                toolbar.multiStateButton.setImage(ImageStop, for: .normal)
-                toolbar.multiStateButton.accessibilityLabel = .TabToolbarStopAccessibilityLabel
-            case .search:
-                toolbar.multiStateButton.setImage(ImageSearch, for: .normal)
-                toolbar.multiStateButton.accessibilityLabel = .TabToolbarSearchAccessibilityLabel
-            case .newTab:
-                toolbar.multiStateButton.setImage(ImageNewTab, for: .normal)
-                toolbar.multiStateButton.accessibilityLabel = .TabToolbarNewTabAccessibilityLabel
```

Comment:
Is it okay to remove these other states, and leave only search and home? @nbhasin2 Based on the answer to this, we'll need to adjust tests in `TabToolbarHelperTests.swift` (`testReloadStateImages` & `testStopStateImages`)

Reply:
yes, totally fine to remove them as its a fixed button
we'll need to re-add them when/if we decide to use multistate on the middle button